test/test_case_fixture.h

```diff
@@ -1904,16 +1932,15 @@ struct test_case_fixture : public base_test_fixture
         std::size_t const batch_size = 9;
         int integers[batch_size] = {1, 2, 3, 4, 5, 6, 7, 8, 9};
         float floats[batch_size] = {1.123f, 2.345f, 3.1f, 4.5f, 5.678f, 6.f, 7.89f, 8.90f, 9.1234f};
-        nanodbc::string::value_type trunc_float[batch_size][6] = {
-            NANODBC_TEXT("1.100"),
-            NANODBC_TEXT("2.300"),
-            NANODBC_TEXT("3.100"),
-            NANODBC_TEXT("4.500"),
-            NANODBC_TEXT("5.700"),
-            NANODBC_TEXT("6.000"),
-            NANODBC_TEXT("7.900"),
-            NANODBC_TEXT("8.900"),
-            NANODBC_TEXT("9.100")};
+        nanodbc::string::value_type trunc_float[batch_size][6] = {NANODBC_TEXT("1.100"),
+                                                                  NANODBC_TEXT("2.300"),
+                                                                  NANODBC_TEXT("3.100"),
+                                                                  NANODBC_TEXT("4.500"),
+                                                                  NANODBC_TEXT("5.700"),
+                                                                  NANODBC_TEXT("6.000"),
+                                                                  NANODBC_TEXT("7.900"),
+                                                                  NANODBC_TEXT("8.900"),
+                                                                  NANODBC_TEXT("9.100")};
```

Comment:
Did you use our `.clang-format`?

Reply:
Used the style script, but from within the container (guess different clang-format version). When I ran the script from the host os, then it changed back these lines.